docs/source/unit-testing.md

```diff
@@ -27,41 +21,35 @@ actors\
     myactor\
         actor.py
         tests\
-            test_mytest.py
+            unit_test.py
 ```
 
 ### Naming conventions
 
 To have the tests found and carried out by
-[pytest framework](https://pytest.org), follow these rules:
-
-- All tests have to reside in the `test_*.py` or `*_test.py` files.
-- Test functions outside of the class have to be prefixed by `test_`.
-- Test methods with the `test_` prefix have to reside in the `Test` prefixed classes.
-- Tests should have unique filenames.
+[pytest framework](https://pytest.org), all test functions have to:
+ - reside in `test_*.py` or `*_test.py` files,
+ - be prefixed by `test_`.
 
-See the pytest [documentation](https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code).
+See the [pytest documentation](https://docs.pytest.org/en/latest/goodpractices.html#tests-outside-application-code).
 
 ### Writing tests that execute the whole actor
 
 Now let's assume you would want to write a test that executes the actor.
-This is how your `test_mytest.py` from above could look like:
+This is how your `unit_test.py` from above could look like:
```

Comment:
just as the run of actor is not unit test, I would rather avoid use of the *unit* word here :)

Reply:
Initially I was thinking to mention here both `unit_test.py` and `component_test.py` to differentiate the two test levels as per https://github.com/oamg/leapp-repository/pull/14. But we can't seem to agree on the naming of the other test level - functional/component/end-to-end: https://github.com/oamg/leapp-repository/pull/14#pullrequestreview-183396803. Till the time we agree on that, I can revert back to `test_mytest.py` here.

Reply:
Reverted.